src/components/Checkout/DonationNudge/DonationNudgeLightboxBorrowerImage.vue

```diff
@@ -57,6 +55,10 @@ import DonationNudgeBoxes from '@/components/Checkout/DonationNudge/DonationNudg
 import DonationNudgeIntro from '@/components/Checkout/DonationNudge/DonationNudgeIntro';
 import KvCharityNavigator from '@/components/Kv/KvCharityNavigator';
 
+// eslint-disable-next-line max-len
+const experimentalMobileImage = require('@/assets/images/checkout/donation-nudge/nudge-borrower-mobile-experimental.jpg');
+const mobileImage = require('@/assets/images/checkout/donation-nudge/nudge-borrower-mobile.jpg');
```

Comment:
@BoulderBrains this is pretty cool, @mcstover found a way we can conditionally insert relative image paths